datasets/ph/sec-advisories/ph_sec_advisories.yaml

```diff
@@ -0,0 +1,52 @@
+title: Philippinies SEC Advisories
+entry_point: crawler.py
+prefix: ph-sec-adv
+coverage:
+  frequency: daily
+  start: 2024-06-26
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Individuals and entities with advisories issued by the Philippines Securities and Exchange Commission
+description: |
+  This dataset comprises information about advisories issued by the
+  Philippines Securities and Exchange Commission (SEC) against individuals
+  and entities. The advisories are issued to warn the public about individuals
+  and entities that are not authorized to solicit investments from the public.
+
+publisher:
+  name: Philippines Securities and Exchange Commission
+  description: |
+    The Securities and Exchange Commission is the agency
+    of the Government of the Philippines charged with the
+    registration and supervision of corporations and
+    securities, as well as capital market institutions
+    and participants
+  url: https://www.sec.gov.ph/#gsc.tab=0
+  official: true
+  country: "ph"
+url: "https://www.sec.gov.ph/investors-education-and-information/advisories/#gsc.tab=0"
+data:
+  url: https://www.sec.gov.ph/investors-education-and-information/advisories/#gsc.tab=0
+  format: HTML
+
+assertions:
+  min:
+    schema_entities:
+      LegalEntity: 500
+  max:
+    schema_entities:
+      LegalEntity: 600
+
+lookups:
+  names:
+    lowercase: true
+    options:
```

Comment:
I actually like this quite a bit!

Reply:
What should I do with the cases we can't identify the name automatically? Do you want me to create a lookup for each one? 

The problem is there seems to be no pattern so I guess we can only do it manually.

For example, on the year 2017 there is some bullet points which are just the name of the company, others that don't have a name and others that have a name plus a explanation (XTRADE.COM is NOT Registered with SEC)
![Screenshot 2024-07-04 at 08 47 58](https://github.com/opensanctions/opensanctions/assets/37581842/d3058797-9b4d-444e-811e-d945a3d995c1)